packages/hdwallet-native/src/fio.ts

```diff
@@ -76,5 +81,33 @@ export function MixinNativeFioWallet<TBase extends core.Constructor<NativeHDWall
         signature: res.signatures[0],
       };
     }
-  };
+
+    async fioEncryptRequestContent(msg: core.FioEncryptionContent): Promise<string> {
+      return this.needsMnemonic(!!this.#mnemonic, async () => {
+        const { fioKey: privateKey } = await fio.FIOSDK.createPrivateKeyMnemonic(this.#mnemonic, core.addressNListToBIP32(msg.addressNList))
+        const cipherAlice = fiojs.createSharedCipher({
```

Comment:
is there something significant about Alice?

Reply:
from example.  will rename